### DISCUSSION TITLE
Is there any way in GrapesJs to implement wireframe like functionality like in figma

### DISCUSSION THREAD

**Discussion by Jagroop-RTE** [GrapesJS/grapesjs#5782] [Q&A]
Is it feasible to develop wire-frame capability in Grapes JS, similar to how Figma allows many pages on the same canvas (body) capabilities?
 OR
 Is there any grapesjs plugin that allows us to implement wireframes?

**NOTE:** I know we can make separate numerous pages, but we want to construct wireframes on the same page (canvas).

**Comment by addlistener** [comment]
Having checked the source code there's only 1 canvas at a time. I don't think its possible. 

But maybe you can wrap multiple gjs canvas together. And zooming may be slow this way since gjs is just DOM and Figma is webgl.
